test/test_cuda_mapper.cc

```diff
@@ -753,6 +753,26 @@ def perforatedConvolution(float(N, C, H, W) input, float(M, C, KH, KW) weights,
   Prepare(tc);
 }
 
+TEST_F(PolyhedralMapperTest, ModulusConstantRHS) {
+  string tc = R"TC(
+def fun(float(N) a) -> (b) { b(i) = a(i % 2) where i in 0:N }
+)TC";
+  // This triggers tc2halide conversion and should not throw.
+  auto scop = Prepare(tc);
+  auto reads = scop->reads;
```

Comment:
hey @ftynse , I have extracted out the reads here but I don't know how to get the isl stride information. I tried to search in the codebase but couldn't find much help. Can you give some pointers? thanks.

Reply:
Yeah, you probably want to search in the third-party/islpp as well :)  There is a greppable documentation file in `doc/`.

Something along the lines of
```c++
for (auto r : scop->reads.range()) {
  if (r.get_tuple_name() != std::string("B")) { // skip irrelevant reads, if any
    continue;
  }
  EXPECT_EQ(r.get_stride(), putExpectedStrideValueHere);
  // EXPECT_TRUE(r.plain_is_universe()); // for the the set being universe
}
```
should work

Reply:
thanks. I tried the above snippet but getting compilation failure on `for (auto r : scop->reads.range())` it seems like it's not iterable. 

is the https://github.com/simbuerg/isl/blob/master/doc/user.pod correct file to consult?

Reply:
Sorry, my bad, try `scop->reads.range().get_set_list()` instead.

This one https://github.com/nicolasvasilache/isl/blob/ntv_dev/doc/user.pod comes from what we submodule (our copy of isl is ahead even of isl's master)